client/src/Movies/Movie.js

```diff
@@ -11,7 +11,8 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id =this.props.match.params.id;
```

Comment:
```suggestion
    const id = this.props.match.params.id;
```
always remember to space properly